src/parking-place/react-app/src/components/Header/PageFunctionButton.tsx

```diff
@@ -52,13 +64,13 @@ function LoginButton() {
           horizontal: "right",
         }}
         open={Boolean(anchorEl)}
-        onClose={handleClose}
+        onClose={handleOpenRepo}
```

Comment:
I think this should remain handleClose
```suggestion
        onClose={handleClose}
```